compose.yaml

```diff
@@ -1,220 +1,95 @@
-# This template is based on https://github.com/frappe/docker-frappe. Modify it to suit your needs.
-# Version: 15
-# Maintainer: Frappe Technologies Pvt. Ltd. <qgJt0@example.com>
+version: "3"
 
 x-custom-image: &custom_image
-  image: ${IMAGE_NAME:-docker.io/frappe/erpnext}:${VERSION:-version-15}
+  image: ghcr.io/jemmia-diamond/jemmia_erp:latest
   pull_policy: ${PULL_POLICY:-always}
   deploy:
     restart_policy:
       condition: always
-
+      
 services:
   backend:
     <<: *custom_image
+    networks:
+      - frappe_network
+    deploy:
+      restart_policy:
+        condition: on-failure
     volumes:
       - sites:/home/frappe/frappe-bench/sites
-    networks:
-      - bench-network
-    healthcheck:
-      test:
-        - CMD
-        - wait-for-it
-        - '0.0.0.0:8000'
-      interval: 2s
-      timeout: 10s
-      retries: 30
+      - logs:/home/frappe/frappe-bench/logs
 
   frontend:
     <<: *custom_image
+    networks:
+      - frappe_network
+    depends_on:
+      - websocket
+    deploy:
+      restart_policy:
+        condition: on-failure
     command:
       - nginx-entrypoint.sh
-    depends_on:
-      backend:
-        condition: service_started
-        required: true
-      websocket:
-        condition: service_started
-        required: true
     environment:
       BACKEND: backend:8000
-      FRAPPE_SITE_NAME_HEADER: ${FRAPPE_SITE_NAME_HEADER:-$$host}
+      FRAPPE_SITE_NAME_HEADER: frontend
```

Comment:
In env, there is a variable  FRAPPE_SITE_NAME_HEADER. What about using it and default host? Now empty maybe later changes